index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <div>
+      <nav class="navbar">
+        <img src="/assets/My Logo.svg" class="logo" />
+        <img class="menu" src="assets/Side.svg" alt="menu" />
+      </nav>
```

Comment:
- [ ] Please the container of the Toolbar section should be a `header` tag as it is semantically better while the hamburger button only should be wrapped in a nav tag.